plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQuerySplit.java

```diff
@@ -31,6 +32,8 @@
 public class BigQuerySplit
         implements ConnectorSplit
 {
+    public static final String VIEW_STREAM = "view-stream";
```

Comment:
Might be useful to name this such that it's clear that this isn't an actual stream that will get created and is just a marker to decide which page source to use.

Maybe `QUERY_SOURCE_MARKER_STREAM`?

Reply:
Or let's have a dedicated field to indicate which page source to use instead of using special stream names to decide that.